src/generic_ui/polymer/settings.ts

```diff
@@ -46,7 +33,7 @@ Polymer({
     this.$.confirmNewServer.hidden = false;
   },
   resetStunServers: function() {
-    browserified_exports.model.globalSettings.stunServers = this.DEFAULT_STUN_SERVERS;
+    browserified_exports.model.globalSettings.stunServers = [];
```

Comment:
Why not reset them to the default?  Won't having no stun servers prevent webrtc from working?